site/source/docs/site/about.rst

```diff
@@ -41,7 +41,7 @@ Building the site
 
 The site sources are stored on `GitHub <https://github.com/emscripten-core/emscripten/tree/incoming/site>`_. Edits and additions should be submitted to this branch in the same way as any other change to the tool.
 
-The site is published to the **emscripten-core/emscripten-site** *gh-pages* branch (GitHub pages).
+The site is published to the **kripken/emscripten-site** *gh-pages* branch (GitHub pages).
```

Comment:
This is the old URL. We don't use `kripken` anymore.  What makes you think we need this change?

Reply:
https://github.com/kripken/emscripten-site/blob/gh-pages/CNAME

Reply:
Not sure how github would handle multiple CNAME files referencing the same domain name. I ended up at kripken/emscripten-site as I failed to find a emscripten-core/emscripten-site repository or a gh-pages branch in emscripten-core/emscripten.